docs/installation_prod.md

```diff
@@ -119,65 +119,65 @@ Cancel the server after testing via `ctrl`+`c`
 
 In order to start up the software as a regular system daemon, similar to a database or webserver, we need to create unit files.
 
-`/etc/systemd/system/adhocracy-plus.service`:
+`/etc/systemd/system/kosmo.service`:
 
 ```
 [Unit]
-Description=adhocracy+ server
+Description=kosmo server
 After=network.target
 
 [Service]
-User=aplus
-WorkingDirectory=/home/aplus/adhocracy-plus
-ExecStart=/home/aplus/.virtualenvs/aplus/bin/gunicorn -e DJANGO_SETTINGS_MODULE=adhocracy-plus.config.settings.production --workers 4 --threads 2 -b 127.0.0.1:8000 -n adhocracy-plus adhocracy-plus.config.wsgi
+User=kosmo
+WorkingDirectory=/home/kosmo/a4-kosmo
+ExecStart=/home/kosmo/.virtualenvs/kosmo/bin/gunicorn -e DJANGO_SETTINGS_MODULE=adhocracy-plus.config.settings.production --workers 4 --threads 2 -b 127.0.0.1:8000 -n kosmo adhocracy-plus.config.wsgi
```

Comment:
here Im not sure what the -n option is? maybe @goapunk can tell me?

Reply:
-n is the process name gunicorn will use (it requires the python setproctitle package):
e.g. without -n it will look like this in the list of running processes:
```
ps ax | grep kosmo
575 ?        Sl     0:09 bin/python3 /home/django/a4_kosmo/bin/gunicorn -e DJANGO_SETTINGS_MODULE=adhocracy-plus.config.settings.production_default --workers 4 --threads 2 -b 127.0.0.1:6000 -n a4_kosmo adhocracy-plus.config.wsgi
```
with -n it looks like this:
```
ps ax | grep kosmo
  12855 ?        S      0:00 gunicorn: master [a4_kosmo]
```

Reply:
Ah cool, thanks for the explanation! So naming it kosmo makes sense I guess?

Reply:
yup, that's fine I'd say